src/reports.coffee

```diff
@@ -131,19 +134,22 @@ plainTemplate = (report) ->
   text = ''
   for data in report.data
     do (data) ->
-      text += " \r\n \r\n <---------- Start Channel  #{data.channel.name} ---------------------------> \r\n \r\n
-                Channel Name: #{data.channel.name} \r\n
-                Channel Load: #{ data.data[0].load } transactions  \r\n
-                Ave response time: #{ data.data[0].avgResp } \r\n
-                Failed:  #{ data.statusData[0].failed }  \r\n
-                Successful:  #{ data.statusData[0].successful }  \r\n
-                Processing: #{ data.statusData[0].processing }  \r\n
-                Completed:  #{ data.statusData[0].completed }  \r\n
-                Completed with errors: #{ data.statusData[0].completedWErrors } \r\n \r\n
-                <---------- End Channel -------------------------------------------------> \r\n \r\n
-              \r\n
-              \r\n
-            "
+      if data.data[0] and data.statusData[0]
+        text += " \r\n \r\n <---------- Start Channel  #{data.channel.name} ---------------------------> \r\n \r\n
+                  Channel Name: #{data.channel.name} \r\n
+                  Channel Load: #{ data.data[0].load } transactions  \r\n
+                  Ave response time: #{ data.data[0].avgResp } \r\n
+                  Failed:  #{ data.statusData[0].failed }  \r\n
+                  Successful:  #{ data.statusData[0].successful }  \r\n
+                  Processing: #{ data.statusData[0].processing }  \r\n
+                  Completed:  #{ data.statusData[0].completed }  \r\n
+                  Completed with errors: #{ data.statusData[0].completedWErrors } \r\n \r\n
+                  <---------- End Channel -------------------------------------------------> \r\n \r\n
+                \r\n
+                \r\n
+              "
+      else
+        text += "There is no data in this"
```

Comment:
Instead of writing there is not data, maybe, we should rather just display values of zero. You may be able to add the null check in the string expressions above. Like `#{ if data.data[0] then data.data[0].load else 0}`